webserver/views/data.py

```diff
@@ -93,7 +93,7 @@ def summary(mbid):
     except db.exceptions.NoDataFoundException:
         summary_data = {}
 
-    if "highlevel" in summary_data:
+    if "highlevel" in summary_data and summary_data["highlevel"]:
```

Comment:
Ah, I see now. This is not the case from the exception catch on line 94.

I'd prefer the `.get()` syntax rather than the double if statement, since an empty dictionary `{}` will evaluate to false.